helper/testhelpers/testhelpers.go

```diff
@@ -911,8 +912,14 @@ func RegisterEntityInTOTPEngine(t testing.T, client *api.Client, entityID, metho
 	if err != nil {
 		t.Fatalf("failed to register a TOTP URL: %v", err)
 	}
+<<<<<<< HEAD
```

Comment:
I'm not sure if you're already aware, but FYI - merge conflicts here!

Reply:
😅

Reply:
We've all done it before! :P